calico_cni/calico_cni.py

```diff
@@ -223,7 +224,7 @@ def delete(self):
         _log.info("Remove networking from container: %s", self.container_id)
 
         # Step 1: Remove any IP assignments.
-        self._release_ip()
+        self._release_ip(self.env)
 
         # Step 2: Get the Calico endpoint for this workload. If it does not
         # exist, log a warning and exit successfully.
```

Comment:
We need to handle errors thrown by `netns.remove_veth` a few lines down from here.